server/services/report/draftInvolvedStaffService.ts

```diff
@@ -0,0 +1,60 @@
+import type DraftReportClient from '../../data/draftReportClient'
+import { StaffDetails } from '../../data/draftReportClientTypes'
+import UserService from '../userService'
+import { Elite2ClientBuilder } from '../../data/elite2ClientBuilder'
+import { AuthClientBuilder } from '../../data/authClientBuilder'
+
+export type DraftInvolvedStaff = StaffDetails & { isReporter?: true }
+export type DraftInvolvedStaffWithPrison = DraftInvolvedStaff & { prison?: string }
+
+export class DraftInvolvedStaffService {
+  constructor(
+    private readonly authClientBuilder: AuthClientBuilder,
+    private readonly elite2ClientBuilder: Elite2ClientBuilder,
+    private readonly draftReportClient: DraftReportClient,
+    private readonly userService: UserService
+  ) {}
+
+  private async reporter(token: string, reporterUsername: string): Promise<DraftInvolvedStaff> {
+    const users = await this.userService.getUsers(token, [reporterUsername])
+    if (!users.length) {
+      throw new Error(`cannot load reporter user: ${reporterUsername}`)
+    }
+    const [user] = users
+    return { ...user, isReporter: true }
```

Comment:
Don't think you need to add an 'isReporter' property here. The function name says it all. In fact the function just retrieves a user. See below.